packages/shared/src/components/cards/PostOptions.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
```

Comment:
I think we can rename this to `OptionsButton`. In our comments box designed by Tsahi, it had the same button, but since we didn't have much action in it yet we had to omit it for now. Though that might mean applying the absolute styling from the consumer's end to make this much more predictable (in the comments box it was in line with the same amount of spacing with other elements).

Reply:
Oke good point will do that :D